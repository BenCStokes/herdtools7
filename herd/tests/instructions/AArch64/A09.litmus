AArch64 A09

(* Tests strb (register), UXTW *)
{ int8_t 0:X0 = 255; 0:X1=x; uint64_t x; 0:X2=0; }

P0;
  STRB W0, [X1, X2, UXTW];

exists (x=255)
