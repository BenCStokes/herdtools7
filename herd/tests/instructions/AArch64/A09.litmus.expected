Test A09 Allowed
States 1
[x]=255;
Ok
Witnesses
Positive: 1 Negative: 0
Condition exists ([x]=255)
Observation A09 Always 1 0
Hash=6267b21bf3ff7747d3301530889f9067

